Polygon.Net/Client/IPolygonClient.cs

```diff
@@ -58,8 +58,24 @@ public interface IPolygonClient
 
         Task<PolygonResponse<TickerType>> GetTickerTypesAsync(string assetClass = default, string locale = default);
 
-        Task<NewsResponse> GetNewsAsync(DateTime? startTime = default, DateTime? endTime = default, string ticker = null, string order = null, int? limit = null, string sort = null);
+        Task<NewsResponse> GetNewsAsync(

```

Comment:
My suggestion is to keep only one and remove the GetTodayNewsAsync and GetNextPageNewsAsync